frameworks/kafka/tests/test_tls.py

```diff
@@ -0,0 +1,100 @@
+import json
+
+import pytest
+
+import sdk_cmd
+import sdk_install
+import sdk_networks
+import sdk_plan
+import sdk_security
+import sdk_utils
+
+
+from tests.test_utils import  *
+
+# Name of the broker TLS vip
+BROKER_TLS_ENDPOINT = 'broker-tls'
+
+
+@pytest.fixture(scope='module')
+def dcos_security_cli():
+    """
+    Installs the dcos enterprise cli.
+    """
+
+    sdk_cmd.run_cli("package install --yes dcos-enterprise-cli")
+
+
+@pytest.fixture(scope='module')
+def service_account(dcos_security_cli):
+    """
+    Creates service account with `hello-world` name and yields the name.
+    """
+    name = 'kafka'
```

Comment:
Essentially all test suites first uninstall and then reinstall the framework. Should be fine.